src/rpdk/core/contract/resource_client.py

```diff
@@ -330,12 +330,8 @@ def get_unique_keys_for_model(self, create_model):
             for k, v in create_model.items()
             if self.is_property_in_path(k, self.primary_identifier_paths)
             or any(
-                map(
-                    lambda additional_identifier_paths, key=k: self.is_property_in_path(
-                        key, additional_identifier_paths
-                    ),
-                    self._additional_identifiers_paths,
-                )
+                self.is_property_in_path(k, additional_identifier_paths)
+                for additional_identifier_paths in self._additional_identifiers_paths
```

Comment:
FYI: This change isn't relevant to the commit, it had to be added due to Flake8 violation with description `C417 Unnecessary use of map - use a generator expression instead.`

Reply:
Addresses 2nd example in the [README here](https://github.com/adamchainz/flake8-comprehensions).